app/api/resources/mentorship_relation.py

```diff
@@ -30,17 +34,31 @@ class SendRequest(Resource):
     def post(cls):
         """
         Creates a new mentorship relation request.
+        Also, sends a notification-like email to the recipient about new relation request.
         """
 
-        user_id = get_jwt_identity()
         data = request.json
+        user_sender_id = get_jwt_identity()
 
         is_valid = SendRequest.is_valid_data(data)
 
         if is_valid != {}:
             return is_valid, 400
 
-        response = DAO.create_mentorship_relation(user_id, data)
+        response = DAO.create_mentorship_relation(user_sender_id, data)
+
+        if user_sender_id == data["mentee_id"]:
+            sender_role = "mentee"
+            user_recipient_id = data["mentor_id"]
+        else:
+            sender_role = "mentor"
+            user_recipient_id = data["mentee_id"]
+
+        user_sender = userDAO.get_user(user_sender_id)
+        user_recipient = userDAO.get_user(user_recipient_id)
+        note = data["notes"]
```

Comment:
```suggestion
        notes = data["notes"]
```
For consistency in the wording and naming of the parameters